oci/private/tarball.sh.tpl

```diff
@@ -17,14 +18,14 @@ readonly OUTPUT="{{output}}"
 function add_to_tar() {
     content=$1
     tar_path=$2
-    echo >>"${OUTPUT}" "${tar_path} uid=0 gid=0 mode=0755 time=1672560000 type=file content=${content}"
+    "${COREUTILS}" echo >>"${OUTPUT}" "${tar_path} uid=0 gid=0 mode=0755 time=1672560000 type=file content=${content}"
```

Comment:
isn't echo shell built-in? i think we don't need coreutils for this.

Reply:
Fair, I will remove that.  I was thinking it wasn't builtin to `sh` but seems I'm foolishly wrong and it's builtin everywhere: https://unix.stackexchange.com/a/1359

Reply:
Updated PR.  Busy day 😅